### DISCUSSION TITLE
How to fix (java.net.SocketException: Connection reset) in LRGS server when connecting to cdadata or cdabackup

### DISCUSSION THREAD

**Discussion by alma660** [opendcs/opendcs#383] [General]
I setup new Redhat 9 servers and loaded OpenDCS-7.0.8. When try to connect to any of the noaa.gov servers I get the java error:

INFO    2023/11/08-16:33:47 DdsRecv Initializing DDS recv connection to cdadata.wcda.noaa.gov:16003
INFO    2023/11/08-16:33:47 DdsRecv Connection established to cdadata.wcda.noaa.gov:16003
WARNING 2023/11/08-16:33:53 DdsRecv:2 Error making connection to cdadata.wcda.noaa.gov:16003: lrgs.lrgsmain.LrgsInputException: IO Error on connection to cdadata.wcda.noaa.gov:16003: java.net.SocketException: Connection reset

Any help on this would be appreciated.
Thanks
Randy

**Comment by MikeNeilson** [comment]
Do you have an account with NOAA for CDADATA? After that i would suggest attempting to connect to the host and port with a telnet, or similar client, just to make sure there's not network issues.

Though "Connection Reset" most likely was caused by a firewall blocking the connection though. with RHEL 9 it's also possible SELinux is on by default and blocking the outbound connections.
